emscripten/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "micropython",
+  "version": "1.8.6",
+  "description": "micropython compiled to ASM.js with emscripten",
+  "main": "main.js",
```

Comment:
`nain` file must point to one that can be `require()`d (importable), use `bin` field and rename the file to `server.js` instead, this would allow to exec it with `npm start` too :-)